.github/labeler-pr-triage.yml

```
@@ -0,0 +1,25 @@
+# PR Labeler documentation: https://github.com/marketplace/actions/labeler
+
+dependencies:
+  - '.github/dependabot.yml'
+  - '**/go.mod'
+  - '**/go.sum'
+documentation:
+  - '**/*.md'
+  - 'docs/**/*'
+  - 'examples/**/*'
+linter:
+  - '.github/workflows/*-lint.yml'
+  - '.golangci.yml'
+provider:
+  - '.go-version'
+  - 'gitlab/**/*'
+  - 'main.go'
```

Maybe we could be a little bit more specific here (which requires additional labels) :)

```suggestion
provider:
  - '.go-version'
  - 'main.go'
  - 'gitlab/provider_*.go'
  
  resource:
  - 'gitlab/resource_*.go'
  
  data-source:
  - 'gitlab/data_source_*.go'
```

---

I like it! Updated. I left the "provider" label globs as-is because I was thinking if it as "if you see this label, then the provider code changed". Thoughts?

---

Yes, sounds good :)